client/script.js

```diff
@@ -0,0 +1,184 @@
+    taskElement.classList.remove('completed');
+  } else {
+    taskElement.classList.add('completed');
+  }
+  
+}
+	
+	function deleteTodo(id){
+	const taskElement = document.querySelector(`li[id='${id}']`);
+	taskElement.remove();
+	}
+	
+	
+	function modifyTodo(event){
+	const button = event.target;
+	const taskid = button.getAttribute('taskid');
+	
+	const todoItem = button.parentNode.parentNode;
+	const todoTitle = todoItem.firstChild.textContent.trim();
+
+  const inputField = document.createElement('input');
+  inputField.type = 'text';
+  inputField.value = todoTitle;
+  const saveButton = document.createElement('button');
+  saveButton.textContent = 'Save';
+  saveButton.addEventListener("click", function (event) {
+    const newTitle = inputField.value.trim();
+    if (newTitle !== '') {
+      todoItem.firstChild.textContent = newTitle;
+      todoItem.removeChild(inputField);
+      todoItem.removeChild(saveButton);
+    } else {
+      alert('Please enter a valid title!');
+    }
+    fetch(`http://localhost:3000/todos/:${todoItem.id}`, {
+      method: "PATCH", 
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({
+        id: +todoItem.id,
+        title: newTitle})
+      }).then(res => {
+          console.log(res.status);
+        })
+    });
+  
+    todoItem.appendChild(inputField);
+    todoItem.appendChild(saveButton);
+	}
+	
+	function removeCompletedTodos() {
+  const completedTodos = document.querySelectorAll('.completed');
+  completedTodos.forEach(todo => todo.remove());
+}
+
+
+document.querySelector('#newTodoContainer button:nth-of-type(1)').addEventListener('click', function () {
+  fetch("http://localhost:3000/todos", {
+      method: "POST", 
+      headers: {
+          "Content-Type": "application/json",
+      },
+      body: JSON.stringify({
+          title: document.getElementById('newTodoInput').value,
+          done: false})
+  }).then(res => {
+    const todoItem = document.getElementById('newTodoInput');
+    const todosContainer = document.querySelector("ul");
+    const liCount = todosContainer.querySelectorAll('li').length;
+    if (todoItem.value.trim() !== '') {
+      let item = {
+        title: todoItem.value.trim(),
+        done: false,
+        id: liCount + 1
+      };
+      addTodo(item);
+      document.getElementById('newTodoInput').value = " ";
+      console.log(res);
+    } else {
+      alert('Please enter a todo title!');
+    }
+  })
+
+});
+
+document.querySelector('#newTodoContainer button:nth-of-type(2)').addEventListener('click', function () {
+fetch("http://localhost:3000/todos", {
+      method: "PUT", 
+      headers: {
+          "Content-Type": "application/json",
+      }
+  }).then(res => {
+      removeCompletedTodos();
+      const todosContainer = document.querySelector('ul'); // Get the <ul> element
+      const todosList = todosContainer.querySelectorAll('li');
+      console.log(res);
+  })
+});
```

Comment:
it's better to place event listeners for "adding new" and "removing completed" buttons into some function. 
like `setup()` or something like that.